src/Microsoft.ML.StandardLearners/Standard/PoissonRegression/PoissonRegression.cs

```diff
@@ -27,7 +26,7 @@
 namespace Microsoft.ML.Runtime.Learners
 {
     /// <include file='doc.xml' path='doc/members/member[@name="PoissonRegression"]/*' />
```

Comment:
>PoissonRegression [](start = 64, length = 17)

If you look in doc.xml in our code sample we have this:
```
new PoissonRegressor
          {
            MaxIterations = 100,
            L2Weight = 0.6f
          }
```
Which is no longer are valid statement.

Reply:
Actually I think it's a much broader issue, and need to be resolved for all converted to estimators classes.

---
In reply to: [220025467](https://github.com/dotnet/machinelearning/pull/957#discussion_r220025467) [](ancestors = 220025467)